src/tables/tableRows.tsx

```diff
@@ -34,6 +34,23 @@ export type Query = {
   activeView?: string | null;
 };
 
+const generateLink = (address?: string | null) =>
+  address ? (
+    <a
+      href={`http://${address}`}
```

Comment:
A lot of  units are running an HTTP server however sometimes it could be custom protocols e.g. `postgres://`, `redis://`,...
Another option could be to show a copy to clipboard button to have the IP in one click and then have the possibility to paste it to a new tab (browser will add http:// prefix automatically) or paste it to program or terminal easily.

https://user-images.githubusercontent.com/36013798/222744307-136a4f10-8e9a-442c-9897-a2016606bd65.mov

Reply:
This is a really good point. I've updated it with a copy button.